cpp/tests/utilities/column_utilities.hpp

```diff
@@ -201,16 +201,16 @@ inline std::pair<thrust::host_vector<std::string>, std::vector<bitmask_type>> to
     cudf::test::expect_column_properties_equivalent(lhs, rhs);  \
   }
 
-#define CUDF_TEST_EXPECT_COLUMNS_EQUAL(lhs, rhs) \
-  {                                              \
-    SCOPED_TRACE(" <--  line of failure\n");     \
-    cudf::test::expect_columns_equal(lhs, rhs);  \
+#define CUDF_TEST_EXPECT_COLUMNS_EQUAL(lhs, rhs...) \
+  {                                                 \
+    SCOPED_TRACE(" <--  line of failure\n");        \
+    cudf::test::expect_columns_equal(lhs, rhs);     \
   }
```

Comment:
https://stackoverflow.com/questions/1067226/c-multi-line-macro-do-while0-vs-scope-block
```suggestion
  do {                                                 \
    SCOPED_TRACE(" <--  line of failure\n");        \
    cudf::test::expect_columns_equal(lhs, rhs);     \
  } while(0)
```

Reply:
+1 for this.